cns/restserver/restserver.go

```diff
@@ -13,6 +13,7 @@ import (
 	"github.com/Azure/azure-container-networking/cns/dockerclient"
 	"github.com/Azure/azure-container-networking/cns/logger"
 	"github.com/Azure/azure-container-networking/cns/networkcontainers"
+	nodesubnet "github.com/Azure/azure-container-networking/cns/nodesubnet"
```

Comment:
Same thing here. I don't think the alias is necessary since the package's name will be "nodesubnet" anyway.

```suggestion
	"github.com/Azure/azure-container-networking/cns/nodesubnet"
```